src/main/java/mate/academy/hibernate/relations/model/Country.java

```diff
@@ -1,6 +1,14 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
 public class Country implements Cloneable {
```

Comment:
Implementing Cloneable and providing a clone method is not necessary for entity classes. Entity classes are typically managed by an ORM framework like Hibernate, which handles their lifecycle, so manual cloning is rarely needed and can lead to confusion and bugs if not handled properly. Consider removing the Cloneable interface and the clone method unless there is a specific requirement for it.